lib/matplotlib/tests/test_colorbar.py

```diff
@@ -771,3 +771,17 @@ def test_inset_colorbar_layout():
     np.testing.assert_allclose(cb.ax.get_position().bounds,
                                [0.87, 0.342, 0.0237, 0.315], atol=0.01)
     assert cb.ax.outer_ax in ax.child_axes
+
+
+@image_comparison(['colorbar_twoslope.png'], remove_text=True,
+                  style='mpl20')
+def test_twoslope_colorbar():
+    # Note that the first tick = 20, and should be in the middle
```

Comment:
Adding the ticklabels to the image would be helpful here.

Reply:
Yeah, I agree, but we really try to avoid labels, because they cause image comparison problems when the font library changes...